addons/main/functions/fnc_3denEventHandlers_onConnectionStart.sqf

```diff
@@ -0,0 +1,9 @@
+params ["_type", "_from"];
+
```

Comment:
Empty file?

Reply:
This file is only there to tell you, that it's not necessary. But I think we can remove it. I will do that.